src/Psalm/Internal/Analyzer/Statements/Expression/IssetAnalyzer.php

```diff
@@ -30,7 +31,16 @@ public static function analyze(
                 $var_id = '$this->' . $isset_var->name->name;
 
                 if (!isset($context->vars_in_scope[$var_id])) {
-                    $context->vars_in_scope[$var_id] = Type::getMixed();
+                    if (isset($context->vars_in_scope['$this'])) {
+                        InstancePropertyFetchAnalyzer::analyze(
+                            $statements_analyzer,
+                            $isset_var,
+                            $context,
+                        );
+                    }
+                    if (!isset($context->vars_in_scope[$var_id])) {
+                        $context->vars_in_scope[$var_id] = Type::getMixed();
+                    }
```

Comment:
The return value of InstancePropertyFetchAnalyzer::analyze is discarded, not sure if/how it needs to be handled.